src/agent/src/mount.rs

```diff
@@ -15,7 +15,7 @@ use std::sync::Arc;
 
 use tokio::sync::Mutex;
```

Comment:
@windniw Could you expand on the commit text, make sure there is at least one line of description 😄 OK to repeat the issue's description.

Reply:
Please also fix your Signed-off-by: to display a real name instead of the `windniw` pseudonym, as required by the DCO process followed by kata (go to the `Developer's Certificate of Origin 1.1` section in https://www.kernel.org/doc/Documentation/process/submitting-patches.rst).